as-http.js

```diff
@@ -64,35 +65,28 @@ TCurlAsHttp.prototype.send = function send() {
 
     function onSent(err, head, stream, body) {
         if (err) {
-            if (self.onResponse) {
```

Comment:
I think we’ll be fine. We don’t use tcurl.exec in Hyperbahn or TChannel—all tests there use the TChannel client directly. We don’t even depend on tcurl in Hyperbahn anymore. There’s a negligible chance that someone else is using it.